File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/MailOrder.scala
package io.swagger.server.model


/**
 * = MailOrder =
 *
 * A mail order record
 *
 * @param id The ID of the order. for example: ''21472''
 * @param status The order status. for example: ''active''
 * @param username The username to use for this order. for example: ''mb21472''
 * @param comment Optional order comment.
 */
case class MailOrder (
  id: Int,
  status: String,
  username: String,
  comment: Option[String]
)

File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/MailOrder.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

/**
 * A mail order record
 */
class MailOrder
{
    /**
     * The ID of the order.
     */
    #[DTA\Data(field: "id")]
    #[DTA\Validator("Scalar", ["type" => "int"])]
    public int|null $id = null;

    /**
     * The order status.
     */
    #[DTA\Data(field: "status")]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $status = null;

    /**
     * The username to use for this order.
     */
    #[DTA\Data(field: "username")]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $username = null;

    /**
     * Optional order comment.
     */
    #[DTA\Data(field: "comment", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $comment = null;

}
